docs/plugin-meta.lua

```diff
@@ -159,6 +160,68 @@ function Channel.by_twitch_id(id) end
 
 -- End src/controllers/plugins/api/ChannelRef.hpp
 
+-- Begin src/controllers/plugins/api/HTTP.hpp
+
+---@class HTTPResult
+---@field data string Data received from the server
```

Comment:
Data is an iffy name for this. `body` or `text` is a more accurate term.

Leaving these fields as methods instead means we could be more flexible when it comes to things like json parsing in the future (see https://docs.rs/reqwest/latest/reqwest/struct.Response.html#implementations)

Reply:
To put methods on it and have data only in C++-land I'd have to rework `HTTPResult` to be a shared resource instead of just a table. I'm not sure if this is worth the complexity of that.